### PR TITLE
check if saved application exists

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "version": "1.0.0",
       "license": "MIT",
```
Comment (swalke-ch) [inline package-lock.json:10]: You don't need to commit this file

File: src/main/controllers/features/common/checkSavedApplicationController.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    let nextPageUrl = "";
+    if (savedApplication.status === 404) {
+        nextPageUrl = addLangToUrl(BASE_URL + TYPE_OF_BUSINESS, lang);
+    } else if (savedApplication.status === 204) {
+        nextPageUrl = addLangToUrl(BASE_URL + SAVED_APPLICATION, lang);
+    } else {
+        logger.error(`internal server error: ` + savedApplication.status);
+    }
+
+    res.redirect(nextPageUrl);
```
Comment (swalke-ch) [inline src/main/controllers/features/common/checkSavedApplicationController.ts:24]: nextPageUrl will be null in case of else condition

**Review by ttingle-ch — COMMENTED** [review]
Could you removed code smells and add a controller unit test for checkSavedApplicationController

File: src/main/controllers/features/common/checkSavedApplicationController.ts (hunk 2 above)
Comment (sardhani-ch) [reply src/main/controllers/features/common/checkSavedApplicationController.ts]: Addressed

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

File: src/main/controllers/features/common/checkSavedApplicationController.ts (hunk 2 above)
Comment (swalke-ch) [reply src/main/controllers/features/common/checkSavedApplicationController.ts]: How will it handled on the UI, I can see error being logged and nothing will be rendered?

**Pull request merged**